Source/HelixToolkit.Wpf/Importers/StudioReader.cs

```diff
@@ -547,7 +547,7 @@ private void ReadMaterial(BinaryReader reader, int chunkSize)
                         }
 
                         mg.Children.Add(new SpecularMaterial(new SolidColorBrush(specular), specularPower));
-
+                        mg.Children.Add(new EmissiveMaterial((new SolidColorBrush( luminance))));

```

Comment:
Ok, the functionality of the wpf 3DS importer should not be broken.
The plan is to move the importers and exporters into the portable library #42.
I prefer your alternative 1.